wizeline/wize-q-front
# Feat - WIP 305 - Assign answer Quickfix

#### What does this PR do?
- This PR fixes the assign-answer PR by putting back the onHomeEnter router call; the store was never getting the admin list.

#### How should this be manually tested?
(List of steps to reproduce, corroborate or tests to run. Write this section clear enough so that external users can also follow it and test the fix.)
1. Login as admin
2. Click on assign on any question
3. Select admin to assign
4. Submit the form and check email if you are the assigned admin

#### What are the relevant tickets?
[Assing questions to admins](https://wizeline.atlassian.net/browse/WIP-305)
